src/Statement/StatementListChanger.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Statement;
+
+use Wikibase\DataModel\Entity\PropertyId;
+use Wikibase\DataModel\Statement\Statement;
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * A collection of functions to manipulate StatementList objects.
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+class StatementListChanger {
```

Comment:
The method names are fine, but the class name is very generic. "Changer" could be anything. The methods mainly relate to groups, so maybe the class should have "Groups" in the name.  But I can't think of anything nice offhand. So whatever.